src/main/java/nl/ordina/jobcrawler/model/Vacancy.java

```diff
@@ -8,7 +8,15 @@
 import nl.ordina.jobcrawler.controller.exception.VacancyURLMalformedException;
 import org.hibernate.annotations.GenericGenerator;
 
-import javax.persistence.*;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
 import javax.validation.constraints.NotNull;
 import java.io.IOException;
 import java.net.HttpURLConnection;
```

Comment:
remove @ AllArgsConstructor as it is already available in @ Data

Reply:
According to [Lombok website](https://projectlombok.org/features/Data) ```@Data``` only includes constructor ```@RequiredArgsConstructor```.

```@AllArgsConstructor``` is needed for the ```@Builder``` annotation, which made me question if this is the right way of doing it since the constructor annotation could have been included in the ```@Builder``` annotation. So if you have another ways of achieving the same functionality please notify me.

Reply:
please leave this as is.